lux-cli/src/lib.rs

```diff
@@ -147,6 +148,12 @@ pub enum Commands {
     /// Interact with the lux configuration.
     #[command(subcommand, arg_required_else_help = true)]
     Config(ConfigCmd),
+    /// Generate autocompletion scripts for the shell.
```

Comment:
suggestion: add an example to the documentation.

```suggestion
    /// Generate autocompletion scripts for the shell.{n}
    /// Example: `lx completion zsh > ~/.zsh/completions/_lx`
```